helm/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
Can we move the chart stuff to `deploy/charts/kubernetes-cluster-prep/` directory? We're going to have multiple charts.